src/digitalmarketplace/components/attachment/template.njk

```diff
@@ -68,9 +71,12 @@
           {%- if params.numberOfPages -%}
             , <span class="dm-attachment__attribute">{{ params.numberOfPages }} {% if params.numberOfPages == 1 %}page{% else %}pages{% endif %}</span>
           {% endif -%}
+          {%- if params.lastUpdated -%}
+            , <span class="dm-attachment__attribute">Last updated: <time datetime="{{ params.lastUpdated.datetime }}">{{ params.lastUpdated.text }}</time></span>
```

Comment:
What is this 😮 ? A new tag? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time

The compatibility table suggests IE support is a no 🤔 I'm guess it just appears as text though?

Reply:
Yup - we've been happily using it on the Supplier Frontend from 4 years ago: https://github.com/alphagov/digitalmarketplace-supplier-frontend/blame/c8fca9349949d19fe187938bae7b911b83c0aced/app/templates/frameworks/_guidance_links.html

If a browser isn't familiar with it, it'll just treat it as a generic inline element. I've tested on Browserstack and it looks fine:

![Screenshot 2020-11-30 at 10 22 52](https://user-images.githubusercontent.com/22524634/100598042-1d4f1e00-32f6-11eb-9f3b-3e6c07d3e0cc.png)

Reply:
Huh, I had no idea 😅 So cool!